packages/commonwealth/server-test.ts

```diff
@@ -101,48 +102,39 @@ const resetServer = (debug = false): Promise<void> => {
         isAdmin: true,
       });
 
-      const nodes = [
-        ['mainnet1.edgewa.re', 'Edgeware Mainnet', null, BalanceType.Substrate],
-        [
-          'https://eth-mainnet.alchemyapi.io/v2/dummy_key',
-          'Ethereum Mainnet',
-          '1',
-        ],
-        [
-          'https://eth-ropsten.alchemyapi.io/v2/dummy_key',
-          'Ropsten Testnet',
-          '3',
-        ],
-        [
-          'https://rpc-osmosis.ecostake.com',
-          'Osmosis',
-          null,
-          BalanceType.Cosmos,
-        ],
-        [
-          'https://cosmos-devnet.herokuapp.com/rpc',
-          'Cosmos SDK v0.46.11 devnet',
-          null,
-          BalanceType.Cosmos,
-          'https://cosmos-devnet.herokuapp.com/lcd/',
-        ],
-      ];
+      const nodes: Record<string, ChainNodeAttributes> = {
+        edgeware: {
+          url: 'mainnet1.edgewa.re',
+          name: 'Edgeware Mainnet',
+          balance_type: BalanceType.Substrate,
+        },
+        ethereum: {
+          url: 'https://eth-mainnet.alchemyapi.io/v2/dummy_key',
+          name: 'Ethereum Mainnet',
+          eth_chain_id: 1,
+          balance_type: BalanceType.Ethereum,
+        },
+        ropsten: {
+          url: 'https://eth-ropsten.alchemyapi.io/v2/dummy_key',
+          name: 'Ropsten Testnet',
+          eth_chain_id: 3,
+          balance_type: BalanceType.Ethereum,
+        },
+        juno: {
```

Comment:
```suggestion
        osmosis: {
```

Not a big deal. I recently changed this to osmosis, since juno went to v1.